srlife/damage.py

```diff
@@ -217,14 +294,14 @@ def single_cycles(self, tube, material, receiver):
 
     # Material point cycle fatigue damage
     Df = self.fatigue_damage(tube, material, receiver)
-    
+
     nc = receiver.days
 
     # This is going to be expensive, but I don't see much way around it
-    return min(self.calculate_max_cycles(self.make_extrapolate(c), 
-      self.make_extrapolate(f), material) for c,f in 
+    return min(self.calculate_max_cycles(self.make_extrapolate(c),
```

Comment:
Did I do this lol?

Reply:
Yes, I haven't touched any part of the code apart from the Weibull and principal stresses